src/WebJobs.Script.Analyzers/WebJobsAttributeAnalyzer.cs

```diff
@@ -2,18 +2,32 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Diagnostics;
+using System;
 using System.Collections.Immutable;
+using System.Reflection;
 
 namespace Microsoft.Azure.Functions.Analyzers
 {
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public class WebJobsAttributeAnalyzer : DiagnosticAnalyzer
     {
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get { return ImmutableArray.Create(DiagnosticDescriptors.IllegalFunctionName); } }
+        JobHostMetadataProvider _tooling;
+
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
+        {
+            get
+            {
+                return ImmutableArray.Create(
+            DiagnosticDescriptors.IllegalFunctionName,
+            DiagnosticDescriptors.BadBindingExpressionSyntax,
+            DiagnosticDescriptors.FailedValidation);
```

Comment:
nit: we should indent this to come after the return i.e. 

```suggestion
                return ImmutableArray.Create(
                    DiagnosticDescriptors.IllegalFunctionName,
                    DiagnosticDescriptors.BadBindingExpressionSyntax,
                    DiagnosticDescriptors.FailedValidation);
```

Reply:
Whoops! Missed that; thank you!